workspaces/global-floating-action-button/plugins/global-floating-action-button/src/components/CustomFab.tsx

```diff
@@ -68,18 +71,20 @@ const FABLabel = ({
           <FabIcon icon={icon} />
         </Typography>
       )}
-    </Typography>
+    </>
   );
 };
 
-export const FAB = ({
+export const CustomFab = ({
   actionButton,
   size,
   className,
+  iconColor,
```

Comment:
instead of adding a new prop to this Component you could add `iconColor` property to the `FloatingActionButton` type